force-app/main/default/lwc/modalCandidateNew/modalCandidateNew.html

```diff
@@ -0,0 +1,55 @@
+<template>
+    <template if:true={isModalOpen}>
+        <section role="dialog" tabindex="-1" aria-modal="true" aria-labelledby="modal-heading-01" class="slds-modal slds-fade-in-open slds-modal_medium">
+            <div class="slds-modal__container">
+                <div class="slds-modal__header">
+                    <h1 id="modal-heading-01" class="slds-modal__title slds-hyphenate" tabindex="-1">New Candidate</h1>
+                </div>
+                <div class="slds-modal__content slds-p-around_medium" id="modal-content-id-1">
+                    <lightning-record-edit-form class="candidate-form" 
+                                                record-id={newCandidateId} 
+                                                object-api-name={candidateApiName} 
+                                                onsubmit={handleCandidateSubmit} 
+                                                onsuccess={handleCandidateSuccess}>
+                        <div class="slds-grid slds-wrap">
+                            <template for:each={candidateFields} for:item="field">
+                                <div key={field} class="slds-col slds-size_1-of-2">
+                                    <template lwc:if={field.showAsterisk}>
+                                        <lightning-input-field field-name={field.name} required></lightning-input-field>
+                                    </template>
```

Comment:
Do we need to check this condition and handle required fields explicitly?

Reply:
Yes, it doesn't get the field with required attribute for some reason although it is a required standard field